firefox-ios/Client/Frontend/Toolbar+URLBar/ToolbarTextField.swift

```diff
@@ -28,6 +28,8 @@ class ToolbarTextField: AutocompleteTextField {
 
     override init(frame: CGRect) {
         super.init(frame: frame)
+        // Disable OS to load AutoFill, since for URL type is not needed.
```

Comment:
Could change AutoFill -> AutoComplete since we use that terminology more in this file

Reply:
Yes sure i'm going to change it